.github/workflows/test.yaml

```diff
@@ -25,13 +25,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes: [default]
+        kubernetes: [v1.20.2]
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with: { go-version: 1.x }
       - run: go mod download
-      - run: ENVTEST_K8S_VERSION="${KUBERNETES#default}" make check-envtest
+      - run: ENVTEST_K8S_VERSION="${KUBERNETES#v1.20.2}" make check-envtest
```

Comment:
🔧 This should stay `default`. It's not obvious, but the `#` is converting `"default"` to `""`. When the variable is empty, the tooling uses its default.